examples/pxScene2d/src/rcvrcore/AppSceneContext.js

```diff
@@ -315,9 +315,13 @@ AppSceneContext.prototype.runScriptInNewVMContext = function (packageUri, module
   try {
     if (!isDuk && !isV8) {
       var requireMethod = function (pkg) {
-        log.message(3, "old use of require not supported: " + pkg);
-        // TODO: remove
-        return requireIt(pkg);
+        if (typeof requireIt === "function") { 
+          // TODO: remove
```

Comment:
Let's log the require message even if requireIt is defined.

Reply:
Okay, I agree, since the case when .pxsceneEnableRequire is defined would replace the 'requireMethod' directly with 'require' at line 336

Reply:
Fixed review comments.